projects/aws/eks-anywhere-packages/Makefile

```diff
@@ -4,21 +4,34 @@ GOLANG_VERSION?="1.17"
 
 REPO=eks-anywhere-packages
 REPO_OWNER=aws
+
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
+GO_MOD_PATHS=. ecrtokenrefresher
+
 IMAGE_COMPONENT=eks-anywhere-packages
-IMAGE_NAMES=$(IMAGE_COMPONENT)
+ECRTOKENREFRESHER_IMAGE_COMPONENT=eks-anywhere-packages/ecrtokenrefresher
+IMAGE_NAMES=eks-anywhere-packages ecrtokenrefresher
 
-BINARY_TARGET_FILES=package-manager
+BINARY_TARGET_FILES=package-manager ecrtokenrefresher
 BASE_IMAGE_NAME=eks-distro-minimal-base
 
+ECRTOKENREFRESHER_ATTRIBUTION_OVERRIDE=ecrtokenrefresher
+ATTRIBUTION_TARGETS=ATTRIBUTION.txt ATTRIBUTION.txt
```

Comment:
this line doesnt seem right, im not sure what you want instead tho? maybe just remove it?

Reply:
Updated per discussion. Thanks!